app/src/main/java/gov/hhs/cdc/trustedintermediary/wrappers/Formatter.java

```diff
@@ -6,7 +6,9 @@
  */
 public interface Formatter {
 
-    <T> T convertToObject(String input, Class<T> clazz) throws FormatterProcessingException;
+    <T> T convertJsonToObject(String input, Class<T> clazz) throws FormatterProcessingException;
+
+    <T> T convertYamlToObject(String input, Class<T> clazz) throws FormatterProcessingException;
 
     String convertToString(Object obj) throws FormatterProcessingException;
```

Comment:
Would you mind renaming this method to `convertToJsonString` since we differentiate between JSON and YAML now.